modules/SofaGuiQt/SofaGuiQtConfig.cmake.in

```diff
@@ -0,0 +1,42 @@
+# CMake package configuration file for the @PROJECT_NAME@ module
+
+@PACKAGE_GUARD@
+@PACKAGE_INIT@
+
+set(SOFAGUIQT_HAVE_QTVIEWER @SOFAGUIQT_HAVE_QTVIEWER@)
+set(SOFAGUIQT_HAVE_QGLVIEWER @SOFAGUIQT_HAVE_QGLVIEWER@)
+set(SOFAGUIQT_HAVE_QT5_CHARTS @SOFAGUIQT_HAVE_QT5_CHARTS@)
+set(SOFAGUIQT_HAVE_QT5_WEBENGINE @SOFAGUIQT_HAVE_QT5_WEBENGINE@)
+set(SOFAGUIQT_HAVE_NODEEDITOR @SOFAGUIQT_HAVE_NODEEDITOR@)
+set(SOFAGUIQT_HAVE_TINYXML @SOFAGUIQT_HAVE_TINYXML@)
+
+
+find_package(SofaBaseVisual QUIET REQUIRED)
+find_package(SofaLoader QUIET REQUIRED)
+find_package(SofaMiscForceField QUIET REQUIRED)
+find_package(SofaGuiCommon QUIET REQUIRED)
+
+if(SOFAGUIQT_HAVE_QTVIEWER)
+    find_package(Qt5 QUIET REQUIRED Core Gui OpenGL)
+    if(SOFAGUIQT_HAVE_QT5_CHARTS)
```

Comment:
Are the charts and the webengine really dependant on the **QtViewer**?

Reply:
hum this is the current behavior in Sofa right now, so I did not really go further...